app/javascript/components/TenantProfilePage.js

```diff
@@ -34,7 +34,31 @@ class TenantProfilePage extends Component {
 
   handleSubmit(e) {
     e.preventDefault();
-    // Todo: change the values in the database
+    fetch("/tenants/update_tenant", {
+      method: 'PATCH',
+      body: JSON.stringify({
+        tenant_id: this.state.tenant_id,
+        name: this.state.name, 
+        email: this.state.email, 
+        landowner_id: this.state.landowner_id, 
+        street_address: this.state.street_address, 
+        city: this.state.city, 
+        zip: this.state.zip, 
+        state: this.state.state
+      })
+    }).then((response) => {return response.json()}).then((data) => {
+      if (data.code == 200){
+        // TODO: add an alert that it worked
+        console.log("successfully saved changes to db")
+      }
+      else {
+        // TODO: add an alert that it failed

```

Comment:
currently plan to make these todo a separate ticket and put them in a separate pr since the notifications don't seem as important